src/main/target/OMNIBUSF7/target.h

```diff
@@ -168,6 +171,7 @@
 #define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
 #define SERIALRX_UART           SERIAL_PORT_USART2
 #define SERIALRX_PROVIDER       SERIALRX_SBUS
+#define ESC_SENSOR_UART         SERIAL_PORT_USART7
```

Comment:
Can you do this in `config.c` instead? I don't think we should be adding more target specific defines a this point (or for consistency, we should be adding them to all targets).